### DISCUSSION TITLE
7x ABB Charger TAC-W22-T-R-0

### DISCUSSION THREAD

**Discussion by McFranky** [lbbrhzn/ocpp#217] [General]
I have successfully integrated an ABB charger. After I integrated the second into the network with the same configuration, it is not recognized. How are the steps to integrate more than one charger?

Error:
Logger: custom_components.ocpp
Source: custom_components/ocpp/api.py:745
Integration: Open Charge Point Protocol (OCPP) (documentation, issues)
First occurred: 12:25:34 (7 occurrences)
Last logged: 12:28:48

Exception occurred: cannot call recv while another coroutine is already waiting for the next message
Traceback (most recent call last):
  File "/config/custom_components/ocpp/api.py", line 184, in on_connect
    await charge_point.reconnect(websocket)
  File "/config/custom_components/ocpp/api.py", line 745, in reconnect
    await super().start()
  File "/usr/local/lib/python3.9/site-packages/ocpp/charge_point.py", line 125, in start
    message = await self._connection.recv()
  File "/usr/local/lib/python3.9/site-packages/websockets/legacy/protocol.py", line 385, in recv
    raise RuntimeError(
RuntimeError: cannot call recv while another coroutine is already waiting for the next messag
![Screenshot 2021-12-01 123624](https://user-images.githubusercontent.com/59682922/144227966-f87b0e4f-8861-434a-a0c9-bb7c503453b2.png)
![Screenshot 2021-12-01 123727](https://user-images.githubusercontent.com/59682922/144227967-100af58b-a459-435d-b404-15d50bc49d3b.png)
![Screenshot 2021-12-01 123657](https://user-images.githubusercontent.com/59682922/144227968-a0b65360-04a0-4489-88a4-14a87c1e04c6.png)
e

**Comment by lbbrhzn** [comment]
Currently only one charger per central server is supported by this integration. You can create another central server that will listen to a different port (e.g. 9001) ad connect the second charger to that. 

How many charger do you want to connect?

**Reply by McFranky** [reply]
Thanks for your answer. It works now...

**Reply by McFranky** [reply]
OK, one question to the Datapoint "maximum_current". With this Datapoint, I think, can I reduce the current for the charger (energymanagment). But this Datapoint is in status "not available". What is the problem?

**Comment by lbbrhzn** [comment]
It is only available if your charger supports the "Smart Charging" profile. You can check this in the value of the 'Features' sensor.
